static.go

```diff
@@ -260,6 +258,14 @@ func ParseStatic(content []byte, opts ParseStaticOptions) (*Static, error) {
 				return
 			},
 		},
+		{
+			File: "transfers.txt",
+			Action: func(file *csv.File) (w []warnings.StaticWarning) {
+				result.Transfers = parseTransfers(file, result.Stops, result.Routes, result.Trips)
+				return
+			},
+			Optional: true,
+		},
```

Comment:
This moved because must parse transfers _after_ routes + trips now (so that they can be used for lookups)